postgres-driver/load_balancer.go

```diff
@@ -95,6 +95,30 @@ func (lb *dbLoadBalancer) toLoadBalancer() *repository.LoadBalancer {
 	}
 }
 
+type dbLoadBalancerJSON struct {
```

Comment:
The PG driver already has the capability of converting from a "DB type" to the types defined in the `repository` package. All these new types seem redundant, and could be removed in favor of some refactoring to allow use of existing types.

Reply:
Sadly I do need to add this new structures, some types change and the JSON tags do too